src/enums/privacyRegime.ts

```diff
@@ -16,5 +16,7 @@ export enum PrivacyRegimeEnum {
   /** Colorado Privacy Act */
   CPA = 'CPA',
   /** Nevada Senate Bill 220 */
-  'NEVADA_SB220' = 'NEVADA_SB220',
+  NEVADA_SB220 = 'NEVADA_SB220',
+  /** New Federal Act on Data Protection */
```

Comment:
we should note applicable regions/governments in this comment

Reply:
Change made. @eligrey Can you re-approve?